ds3_java_cli/src/main/java/com/spectralogic/ds3cli/exceptions/TapeFailureException.java

```diff
@@ -1,69 +0,0 @@
-/*
```

Comment:
It wasn't used anywhere.  Initially it was used when we added the check prior to doing a bulk operation to see if there were any failures, but when we pulled that out, this code was no longer being used.